components/kubeflow-pipeline/pipeline_components/analytics-server/pyproject.toml

```
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "analytics-server"
+version = "0.1.0"
+description = ""
+authors = ["Jan Schlicht <jan@katulu.io>"]
```

Let's use:

```suggestion
authors = ["Katulu GmbH <eng@katulu.io>"]
```

Instead